sdk/attestation/service.projects

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- Remove data plane tests from nightly live runs: https://github.com/Azure/azure-sdk-for-net/issues/10437 -->
+    <ProjectReference Remove="$(MSBuildThisFileDirectory)Microsoft.Azure.Attestation\**\*.csproj" Condition="'$(AZURE_SEARCH_TEST_MODE)' == 'Live'" />
```

Comment:
That conditions isn't ever set for your projects. I'd just remove the condition entirely to remove those projects in all contexts.